source/includes/_endpoint-dataset.md

```diff
@@ -486,6 +486,16 @@ in the API.
 
 Categorical variables will be exported with their name instead of their value.
 
```

Comment:
Above, L456: this is the first shoji:view I've encountered that does not have its content inside a "value" attribute (here it is "views"). According to the [shoji specification](https://bitbucket.org/fumanchu/shoji/src/tip/spec.txt?fileviewer=file-view-default#spec.txt-892), it should be under "value".